src/showcase/app/journey-maps/journey-maps.module.ts

```diff
@@ -1,5 +1,5 @@
 import { PortalModule } from '@angular/cdk/portal';
-import { CommonModule } from '@angular/common';
+import { CommonModule, DecimalPipe } from '@angular/common';
```

Comment:
DecimalPipe is already provided by CommonModule: https://github.com/angular/angular/blob/15.1.4/packages/common/src/pipes/index.ts#L55
This change is unnecessary.

Reply:
It still has to be listed as a provider. The following error occurs when it is not given

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/83814947/223073985-f9625299-7f53-4223-8dc8-7879e86ec51a.png">